scripts/qa_crowbarsetup.sh

```diff
@@ -3334,6 +3335,7 @@ function deploy_single_proposal()
             ;;
         trove)
             iscloudver 5plus || return
+            [[ $want_trove = 1 ]] || return
```

Comment:
@jdsn Thanks for putting some more meat into my question :) I always get confused with the style used above until I remember that it's a `||` and not an `&&`.